README.md

```diff
@@ -1 +1,66 @@
-# Microverse-Tic-Tac-Toe-Ruby
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Tic Tac Toe
+
+In order to win the game, a player must place three of their marks in a horizontal, vertical, or diagonal row.
+Players will soon discover that the best play from both parties leads to a draw. 
+
+## How do we play Tic Tac Toe?
+
+- Run main on your terminal
```

Comment:
You need to tell exactly how to start the game. You tell the player that they must run `main` to start the game but you don't ask if they are on the `bin` directory. Tell the player step by step how to start the game. :hammer_and_wrench: 
[OPTIONAL]  For me it is better to delete this line, in this section you are explaining how to play tic-tac-toe in general, you are not telling how to play your game. You already are explaining how to start your game in (Getting Started)